### DISCUSSION TITLE
Transition from Mask to Pattern should be in breaking changes list

### DISCUSSION THREAD

**Discussion by pekka** [filamentphp/filament#7498] [General]
It appears that the use of `Filament\Forms\Components\TextInput\Mask` has been deprecated in v3 in favour of a new class named `Filament\Forms\Components\TextInput\Pattern`.

That should probably be on the list of breaking changes.

Should I submit a PR? I wouldn't be able to talk much about `Pattern` though.

**Comment by danharrin** [comment]
There is no class named Pattern, not sure where you got that from

**Reply by pekka** [reply]
Huh. Let me check where I got that from. Sounds like an AI hallucinating. I'm sure I wasn't  😂 my apologies.
